tasklib/tests.py

```diff
@@ -839,16 +843,17 @@ class TaskFromHookTest(TasklibTest):
         '"status":"pending",'
         '"uuid":"81305335-0237-49ff-8e87-b3cdc2369cec"}')
 
-    input_modify_data = six.StringIO(input_add_data.getvalue() + '\n' +
+    input_modify_data = six.StringIO(
+        input_add_data.getvalue() + '\n' +
```

Comment:
In this case, we could use `str.join` to make the code a bit clearer:

```python
input_modify_data = six.StringIO(
    '\n'.join([
        input_add_data.getvalue(),
        (
            '{"description":"Buy some milk finally",'
            '"entry":"20141118T050231Z",'
            '"status":"completed",'
            '"uuid":"a360fc44-315c-4366-b70c-ea7e7520b749"}'
        ),
)

Reply:
I specially liked this example, thanks!

Fixed in 8f4fabf